xml_converter/generators/code_generator.py

```diff
@@ -267,6 +267,35 @@ def load_input_doc(self, dir_path: str) -> None:
                 content=document.content
             )
     
+    def write_parseable(self, output_directory: str) -> None:
+        file_loader = FileSystemLoader('cpp_templates')
+        env = Environment(loader=file_loader)
+        template = env.get_template("parseabletemplate.hpp")
+        attributenames: Dict[str,str] = {}
+        
+        for filepath in self.data.keys():
+            filename = os.path.basename(filepath)
+            attributenames[filepath]= filename.split(".md")[0]
+     
+        pages = ["Category","Icon","Trail"]
+        
+        for page in pages:
+            content: Dict[str,str] = {}
+            metadata: Dict[str,List[str]] = {}
+            
+            for attributename in attributenames:
+                metadata[attributename] = self.data[attributename].metadata
+            
+            field_rows, includelist = self.generate_node_types(metadata, attributenames, page)
```

Comment:
Lets change `field_rows` and  `includelist` to mach whatever the new variable name from `generate_node_types()` is.

Reply:
Now `attribute_variables, cpp_include_paths`